### PR TITLE
Feature/#12 add rails admin config

### LINKED ISSUE #12: Add rails admin
## REQUIREMENTS / DESCRIPTION

> Add and config rails admin

### PR DESCRIPTION
**Pull request by Alonso2105**
Description
What did you implement/Why did you implement this?
Configured rails admin to have an admin view, using the 'rails_admin' gem
we create the necessary migrations to add references between the tables
Created the necessary fixes for the tests to work

## Screenshots
Screenshot 2023-08-02 at 20 12 07
Screenshot 2023-08-02 at 21 08 33
Screenshot 2023-08-02 at 21 08 39

### CODE REVIEW

File: shipments_tracker/config/initializers/rails_admin.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  #         ]
+  #       end
+  #     end
+  #     field :carrier_id, :enum do
+  #       enum do
+  #         Carrier.all.map { |carrier| [carrier.name, carrier.id] }
+  #       end
+  #     end
+  #   end
+  # end
```
Comment (JuanequeX) [inline shipments_tracker/config/initializers/rails_admin.rb]: if this code is unnecessary please remove it

File: shipments_tracker/config/initializers/rails_admin.rb [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  ]
+
+
+  config.model 'Shipment' do
+    edit do
+      # field :carrier_id, :enum do
+      #   enum do
+      #   Carrier.all.map { |carrier| [carrier.name, carrier.id] }
+      #   end
+      # end
```
Comment (JuanequeX) [inline shipments_tracker/config/initializers/rails_admin.rb]: remove this comment code

File: shipments_tracker/config/importmap.rails_admin.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+pin "rails_admin/src/rails_admin/base", to: "https://ga.jspm.io/npm:rails_admin@3.1.2/src/rails_admin/base.js"
+pin "@hotwired/turbo", to: "https://ga.jspm.io/npm:@hotwired/turbo@7.3.0/dist/turbo.es2017-esm.js"
+pin "@hotwired/turbo-rails", to: "https://ga.jspm.io/npm:@hotwired/turbo-rails@7.3.0/app/javascript/turbo/index.js"
+pin "@popperjs/core", to: "https://ga.jspm.io/npm:@popperjs/core@2.11.8/dist/esm/popper.js"
+pin "@rails/actioncable/src", to: "https://ga.jspm.io/npm:@rails/actioncable@7.0.6/src/index.js"
+pin "@rails/ujs", to: "https://ga.jspm.io/npm:@rails/ujs@6.1.7/lib/assets/compiled/rails-ujs.js"
+pin "bootstrap", to: "https://ga.jspm.io/npm:bootstrap@5.3.1/dist/js/bootstrap.esm.js"
+pin "flatpickr", to: "https://ga.jspm.io/npm:flatpickr@4.6.13/dist/esm/index.js"
+pin "jquery", to: "https://ga.jspm.io/npm:jquery@3.7.0/dist/jquery.js"
+pin "jquery-ui/", to: "https://ga.jspm.io/npm:jquery-ui@1.13.2/"
```
Comment (JuanequeX) [inline shipments_tracker/config/importmap.rails_admin.rb:11]: add a line at the end of this file

File: shipments_tracker/bin/dev [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+exec foreman start -f Procfile.dev "$@"
```
Comment (JuanequeX) [inline shipments_tracker/bin/dev:8]: FIY: in my current project we have this configuration and the performance running it with `bin/dev` is excellent, this is only a suggestion, if it is useful for you apply it, if not forget about it
```
#!/usr/bin/env sh
OVERMIND_SKIP_ENV=1
export OVERMIND_SKIP_ENV

PORT="${PORT:-3000}"
export PORT

if command -v overmind > /dev/null 2>&1; then
  exec overmind start -f Procfile.dev "$@"
else
  echo "Installing foreman..."
  gem install foreman
  exec foreman start -f Procfile.dev "$@"
fi
```

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: shipments_tracker/config/importmap.rails_admin.rb (hunk 3 above)
Comment (Alonso2105) [reply shipments_tracker/config/importmap.rails_admin.rb:11]: It's done

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**